docs/user-guide/src/tour/summary.md

```diff
@@ -38,15 +36,16 @@ devices.
 As a quick reference, the main steps of this tour and the involved Prusti features
 are as follows:
 
-1. [Getting Started](getting-started.md): Simple runtime errors caught by Prusti
-2. [New](new.md): Postconditions, pure functions
-3. [Push](push.md): Preconditions, trusted functions, old expressions, quantifiers
-4. [Pop](pop.md): Exercise with similar concepts as for push
-5. [Testing](testing.md): More runtime errors caught by Prusti
-6. [A Bad Stack](bad-stack.md): Wrap-up of the second chapter of
-   [Learn Rust with Entirely Too Many Linked Lists](https://rust-unofficial.github.io/too-many-lists/).
-7. [Options](options.md): Verification with option types
-7. [Generics](generics.md): Prusti and generics
-8. [Peek](peek.md): Exercise
-9. [Final Code](final.md): Final code with solution of exercise
-10. [Pledges](pledges.md): Bonus demonstrating Prusti's pledges
+1. [Setup](setup.md) How to add Prusti to a Rust project
+2. [Getting Started](getting-started.md): Simple runtime errors caught by Prusti
+3. [New](new.md): Postconditions, pure functions
+4. [Push](push.md): Preconditions, external specifications, trusted functions, old expressions, quantifiers, snapshots, structural/snapshot equality
+5. [Pop](pop.md): Similar concepts as in [Push](push.md), predicates
+6. [Testing](testing.md): Showing guarantees of verification vs running tests, and how to test specifications
+7. [Option](option.md): Changing `Link` to use `Option` type
+8. [Generics](generics.md): Prusti and generics
+9.  [Peek](peek.md): Verifying a `peek` function
+10. [Pledges (mutable peek)](pledges.md): Demonstrate Prusti's pledges for functions returning mutable references
+11.  [Final Code](final.md): Final code for the verified linked list
+12.  [Loop Invariants](loop_invariants.md): Verifying code containing loops by writing loop invariants
+13.  [Counterexamples](counterexamples.md): Get a counterexample for a failing assertions
```

Comment:
This and pledges: `Getting a counterexample…` and `Demonstrating Prusti's pledges…`, respectively, to be consistent with the other items. Setup can also say `Adding Prusti to…`.